pkg/virt-controller/services/nodeselectorrenderer.go

```diff
@@ -43,12 +46,20 @@ func (nsr *NodeSelectorRenderer) Render() map[string]string {
 		nodeSelectors[cpuFeatureLabel] = "true"
 	}
 
+	if nsr.isTSCSchedulable() {
+		nodeSelectors[topology.ToTSCSchedulableLabel(*nsr.tscFrequency)] = "true"
+	}
+
 	for k, v := range nsr.userProvidedSelectors {
 		nodeSelectors[k] = v
 	}
 	return nodeSelectors
 }
 
+func (nsr *NodeSelectorRenderer) isTSCSchedulable() bool {
```

Comment:
nit: I find this name a bit unclear. What do you think about something like `requresManualTSCFrequency`?

Reply:
`isManualTSCFrequencyRequired` also works imo :)

Reply:
That's settled then. I'll rename it accordingly.

Reply:
Done.